myriad-scheduler/src/test/resources/myriad-config-test-default-with-docker-info.yml

```diff
@@ -20,10 +20,10 @@ checkpoint: false
 frameworkFailoverTimeout: 43200000
 frameworkName: MyriadTest
 frameworkRole: test
-frameworkUser: hduser     # User the Node Manager runs as, required if nodeManagerURI set, otherwise defaults to the user
+frameworkUser: hduser     # User the Node Manager runs as, required if nodemanagerURI set, otherwise defaults to the user
                           # running the resource manager.
 frameworkSuperUser: root  # To be depricated, currently permissions need set by a superuser due to Mesos-1790.  Must be
-                          # root or have passwordless sudo. Required if nodeManagerURI set, ignored otherwise.
+                          # root or have passwordless sudo. Required if nodemanagerURI set, ignored otherwise.
```

Comment:
Looks like int MyriadExecutorConfiguration it's nodeManagerURI not nodemanagerURI, I like it better your way but we should be careful making configuration changes.